pepper-apis/dss-database/src/main/resources/db-changes/schema/DDP-8631-add-country-field-to-medical-provider.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                   http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+
+    <changeSet author="gskhirel" id="physician_country_add">
+        <addColumn tableName="user_medical_provider">
+            <column name="country" type="varchar(100)" />
```

Comment:
Let's add a constraint to explicitly make this nullable (Liquibase's default is database-dependent).